modules/examples/example-tf/main.tf

```diff
@@ -0,0 +1,29 @@
+# Provider Configuration
+provider "aws" {
+  region = var.aws_region
+  access_key = ""
+  secret_key = ""
```

Comment:
We can drop these right? You don't need them afaik?

Reply:
which ones you meant?